### ISSUE TITLE
S110 Inherit System.Windows.Forms.Form

### ISSUE THREAD

**Issue by nnpcYvIVl** [SonarSource/sonar-csharp#245]
Creating the following classes triggers S110 'This class has 7 parents which is greater than the 5 authorized.'

Shouldn't this rule be more lenient for built-in .NET classes? With the current implementation it is not possible to inherit many common controls. 

```csharp
public class NewForm : Form { }
public class NewUserControl : UserControl { }
```

**Comment by Evangelink** [comment]
Hi @nnpcYvIVl !

Thanks for reporting this. We are already aware and have already fixed the [issue](https://github.com/SonarSource/sonar-csharp/issues/231). This will be shipped with the version 5.10.

Cheers,
DotNet Team
